Add HOWTO and scripts for documentation videos (#396)
iiasa/message_ix: doc/video/install.rst

Video script: Installation
**************************

Introduction
============

.. note::

   Use a totally fresh install of Anaconda.
   This includes removing conda-forge from the channels list, so there is no warning message in Step 6.

   Open the install documentation at https://docs.messageix.org.
   Open https://github.com/iiasa/message_ix in a separate tab, for later.

Hello everybody and welcome to this video tutorial on the installation of MESSAGEix through Anaconda on Windows.

> 

Reviewer: ```suggestion
This video has three parts. First, we will show you how to install MESSAGEix. Then, we will run you through checking that your installation was successful. And last, we'll show you how to download and run tutorial model files. You can jump to each part using the chapter markers below.
```

> Here, we are looking at the Installation page of the MESSAGEix documentation at docs dot messageix dot org.

Reviewer: ```suggestion
You can also find on the Installation page of the MESSAGEix documentation at docs dot messageix dot org, shown here. It is also linked in the description below. While you are there, make sure to check the prerequisite knowledge and skills to use MESSAGEix, as this will not be covered in our tutorials.

That being said, let's jump straight into installing MESSAGEix through Anaconda
```

At this point, we have already completed Steps 1 through 3, to install GAMS and add its PATH environment variable on the system.
We have also completed Step 4, installing Anaconda using the instructions linked from this page.
If you haven't completed these Steps 1 through 4 yet, you should read these instructions for those steps, complete them—then come back to this video.

We begin with Step 5, opening the Anaconda Prompt from the Start Menu.

.. note::

   Do this.

> In Step 6, we will make sure that “conda forge” is used as the default channel to download and install packages, including message_ix and other packages that it requires.

Reviewer 2: ```suggestion
In Step 6, we will first make sure that “conda forge” is used as the default channel to download and install packages, including message_ix and other packages that it requires. Secondly, we set the channel priority to 'strict'. This ensures that conda will select the latest version of message_ix from the "noarch" channel and ignore much older packages in OS-specific channels.

```

> We copy this command into the prompt and run it.

Reviewer 2: ```suggestion
We copy the first command into the prompt and run it.
```

.. note::

   Do this.

> The output of the command shows us that this channel is now the top-priority or default channel.

Reviewer 2: ```suggestion
The output of the command shows us that this channel is now the top-priority or default channel.

We then copy the second command into the prompt and run it.

.. note::

   Do this.

Now the channel priority is set to strict.
```

The first step is to download the tutorial files.
Since these are a kind of learning aid, they are not automatically installed with the Python and GAMS code for MESSAGEix.
This is why we need to download them.

The "message-ix" program we already used has a command "dl" that does this for us.
We only need to choose a specific folder or directory where we want the downloaded tutorials to be placed.
In this case, let's put them on the Desktop—but they could also be placed anywhere else.
We use Windows Explorer to navigate to the Desktop, and then copy the full path.

.. note::

   Do this.

Next, with that same "ws1" Anaconda environment active, we run the command "message-ix dl " and then paste the path we just copied.

.. note::

    Do this.

We see that it retrieves some data and unzips it into that specific folder.
It's a very quick process, so we already see that this folder has appeared, and if we double-click on it and then we will find this “tutorial” folder, containing all the different tutorials.

.. note::

   Do this.

The tutorials are in the form of Jupyter notebooks.

> Understanding and working with Jupyter notebooks is one of the prerequisite skills for learning MESSAGEix, these are listed in the documentation.

Reviewer: ```suggestion
```
Covered in the beginning

> This video does not cover this.

Reviewer: ```suggestion
```

> We want to start the Jupyter notebook server and use this to open the tutorials.

Reviewer: ```suggestion
As such, we want to start the Jupyter notebook server and use this to open the tutorials.
```

Video script: Installation
**************************

Introduction
============

.. note::

   Use a totally fresh install of Anaconda.
   This includes removing conda-forge from the channels list, so there is no warning message in Step 6.

   Open the install documentation at https://docs.messageix.org.
   Open https://github.com/iiasa/message_ix in a separate tab, for later.

Hello everybody and welcome to this video tutorial on the installation of MESSAGEix through Anaconda on Windows.

> You can also find on the Installation page of the MESSAGEix documentation at docs dot messageix dot org, shown here. It is also linked in the description below. While you are there, make sure to check the prerequisite knowledge and skills to use MESSAGEix, as this will not be covered in our tutorials.

Reviewer 2: ```suggestion
You can also find the installation of MESSAGEix through Anaconda on Windows on the installation page of the MESSAGEix documentation at docs dot messageix dot org, shown here. It is also linked in the description below. 
While you are there, make sure to check the prerequisite knowledge and skills to use MESSAGEix, as this will not be covered in our tutorials.
```
